tests/regression/test_star_pc.py

```diff
@@ -168,6 +165,8 @@ def test_star_equivalence(problem_type, backend):
                        "mg_coarse_assembled_pc_type": "lu"}
 
     star_params["mg_levels_pc_star_backend"] = backend
+    star_params["mg_levels_pc_star_mat_ordering_type"] = "rcm"
+    star_params["mg_levels_pc_star_sub_sub_pc_factor_mat_ordering_type"] = "natural"
```

Comment:
I had to think a bit to understand what's going on here (that when we specify a `pc_star_mat_ordering_type`, we don't want PETSc's defaults to screw it up later on -- right?).  Is this the best solution?  Could we either default to `sub_pc_factor_mat_ordering_type="natural"` when we set `sub_pc_type` to `lu` in the class defaults, or set this if we find `pc_star_mat_ordering_type` to be set in the options?

Reply:
I am more inclined to have `sub_pc_factor_mat_ordering_type="natural"` as default, regardless of whether or not we set `pc_star_mat_ordering_type`. If no ordering is set then the DOFs in the patches get sorted in the local ordering, I think this should keep the ordering by fields.

Reply:
I'm quite happy with that - it makes sense, and can always be overridden by a knowledgable user.